### PR TITLE
Governance launch prep

### PR DESCRIPTION
**Pull request by dbrehmer**
I think the process is sorted out here for the launch and configure sequence.
Governance still needs the method to add and fund a market, but otherwise it is ready to go.
The only market it will launch with is the sICX/ICX market. Then we will need to fund that one and set up sICX/bnUSD right after launch. Then BALN/bnUSD the second day. I'm thinking we could have one method that funds and sets up a market by name, then we call that from the notebook.

This PR has a lot of small changes since I went through every SCORE and cleaned them up. Most are fairly inconsequential, but let me know if there is anything you'd like to discuss.

### CODE REVIEW

**Review by dbrehmer — COMMENTED** [review]
Looks good to me.

File: core_contracts/governance/governance.py [hunk 1]
```diff
@@ -91,11 +112,6 @@ def _set_launch_time(self, _day: int) -> None:
     def getLaunchTime(self) -> int:
         return self._launch_time.get()
 
-    @external(readonly=True)
```
Comment (ajaya-man) [inline core_contracts/governance/governance.py:94]: Aren't there any other scores having dependency on this method?

**Review by Suyog007 — COMMENTED** [review]
looks good

File: core_contracts/governance/governance.py (hunk 1 above)
Comment (dbrehmer) [reply core_contracts/governance/governance.py:94]: No dependence on that. It is for informational purposes only. The time offset is set in Loans and DEX during the launchBalanced method call.

**Pull request merged**